packages/react/src/components/ActionList/ActionList.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {
```

Comment:
I noticed Cauldron doesn't have coverage checks setup - not for this PR but we should add that

https://github.com/dequelabs/cauldron/issues/1918

Reply:
Ignore - https://github.com/dequelabs/cauldron/issues/1918#issuecomment-2856816680